scripts/deploy/aks-install-mayastor.sh

```diff
@@ -19,13 +19,8 @@ readonly PKG_ROOT="$(git rev-parse --show-toplevel)"
 
 kubectl label nodes --selector agentpool=agentpool openebs.io/engine=mayastor --overwrite
 
-kubectl apply -f "${PKG_ROOT}/scripts/deploy/actions/hugepage-enabler-daemonset.yaml"
-kubectl rollout status daemonset/hugepage --watch --timeout 5m
-kubectl apply -f "${PKG_ROOT}/scripts/deploy/actions/kured-config.yaml"
-sleep 15m
-kubectl delete -f "${PKG_ROOT}/scripts/deploy/actions/kured-config.yaml" --ignore-not-found=true
-kubectl delete -f "${PKG_ROOT}/scripts/deploy/actions/hugepage-enabler-daemonset.yaml" --ignore-not-found=true
-sleep 2m
+kubectl apply -f "${PKG_ROOT}/scripts/deploy/actions/azstor-prereq.yaml"
+kubectl rollout status daemonset/azstor-prereq --watch --timeout 5m
```

Comment:
Should we delete this daemonset as soon as the hugepages is enabled, instead of deleting this with uninstall? Since this is required only as a pre req, I'm assuming we don't need to keep it running?

Reply:
We need to keep it running to cover for the cases such as node failovers and node pool scale out (like we do in Aurora). We are using a file marker to make sure we don't restart repeatedly. It's less that ideal, but I think we will have to live with this for now.